### PR TITLE
[Annotation Processor] Set default int type to BigInteger

### PR DESCRIPTION
**Pull request by satran004**
- Default Int type to `BigInteger`
- Package name to lowercase

#422

### CODE REVIEW

File: annotation-processor/src/main/java/com/bloxbean/cardano/client/plutus/annotation/processor/blueprint/BlueprintAnnotationProcessor.java [hunk 1]
```diff
@@ -81,6 +81,7 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
             try {
                 plutusContractBlueprint = PlutusBlueprintLoader.loadBlueprint(blueprintFile);
             } catch (Exception e) {
+                e.printStackTrace();
```
Comment (nemo83) [inline annotation-processor/src/main/java/com/bloxbean/cardano/client/plutus/annotation/processor/blueprint/BlueprintAnnotationProcessor.java:84]: is this necessary?

File: annotation-processor/src/main/java/com/bloxbean/cardano/client/plutus/annotation/processor/blueprint/DataTypeProcessUtil.java [hunk 2]
```diff
@@ -207,6 +208,8 @@ public FieldSpec processPlutusDataType(String javaDoc, BlueprintSchema schema, S
     private String getPackage(String ns) {
         String pkg = (ns != null && !ns.isEmpty())? annotation.packageName() + "." + ns + ".model"
                 : annotation.packageName() + ".model";
+
+        pkg = pkg.toLowerCase();
```
Comment (nemo83) [inline annotation-processor/src/main/java/com/bloxbean/cardano/client/plutus/annotation/processor/blueprint/DataTypeProcessUtil.java:212]: wondering if we should also remove `-` and `_` ?

**Review by nemo83 — APPROVED** [review]
LGTM, to minor comments

File: annotation-processor/src/main/java/com/bloxbean/cardano/client/plutus/annotation/processor/blueprint/BlueprintAnnotationProcessor.java (hunk 1 above)
Comment (satran004) [reply annotation-processor/src/main/java/com/bloxbean/cardano/client/plutus/annotation/processor/blueprint/BlueprintAnnotationProcessor.java:84]: Let's keep this for now till we find a better way to notify error during compilation.

File: annotation-processor/src/main/java/com/bloxbean/cardano/client/plutus/annotation/processor/blueprint/DataTypeProcessUtil.java (hunk 2 above)
Comment (satran004) [reply annotation-processor/src/main/java/com/bloxbean/cardano/client/plutus/annotation/processor/blueprint/DataTypeProcessUtil.java]: Thanks @nemo83 . Done.

**Pull request merged**